web-ng/src/app/components/inline-editor/inline-editor.component.scss

```diff
@@ -14,12 +14,33 @@
  * limitations under the License.
  */
 
+@import 'variables';
+
 :host {
-  display: block;
-  font-size: 1.5em;
-  margin-top: 0.83em;
-  margin-bottom: 0.83em;
-  margin-left: -8px;
-  margin-right: 0;
   font-weight: bold;
+  font-size: 14pt;
+}
+
+// Share styling of inline edit with inline-edit-title and inline-editor component.
+.inline-editor-input {
```

Comment:
I believe this class is now the same as `:host`, so this style rule can be merged into `:host` and removed.

Reply:
Sorry this doesn't work as host is for parent container but we are trying to give styling to input specifically